### ISSUE TITLE
[Bug] Viz Shonen Jump: Manga thumbnails not loading

### ISSUE THREAD

**Issue by iamkroot** [inorichi/tachiyomi-extensions#3567]
Thanks @alessandrojean @TheUnlocked for your efforts! I just installed the extension, and the chapters seem to be loading fine. However, none of the manga thumbnails are loaded:
<img alt="Viz thumbs not loaded" src="https://user-images.githubusercontent.com/29517049/85108957-370c1580-b22e-11ea-8e96-6129ba131fb6.jpg" height="480px"></img>

### Device information
- Tachiyomi version: r2421
- Android version: 10

### Source information
- Name: Viz Shonen Jump
- Extension version: 1.2.1

**Comment by TheUnlocked** [comment]
I'm seeing this too. For some reason the development environment wasn't having the same issues as the actual build, though after cleaning the dev app I get the same issue. Should be an easy fix, though I'll test it on my actual phone before making a PR.

**Comment by TheUnlocked** [comment]
The PR has been made. Note that even after it updates, the thumbnails still might not fix themselves. I've found that they can be quite stubborn with updating, so you might end up needing to experiment with purging data in the advanced settings to get it to work.

**Comment by iamkroot** [comment]
You weren't kidding about them being stubborn lol. I'd already migrated some manga to Viz with broken thumbs, and now they seem to be cached a bit too well. I migrated back to shueisha, cleared all cache from the advanced settings. When I refreshed the Viz ext page, the thumbs were loading fine. So I migrated to Viz again, but alas - it showed broken thumbs in the library.

Seems like I'll have to do a full "clear data" from android settings
